src/sentry/analytics/event.py

```diff
@@ -27,7 +27,7 @@ def __init__(
     ) -> None:
         self.uuid = uuid1()
         self.datetime = datetime or timezone.now()
-        self.type = self._get_type(type)
+        self.type = self._get_type(type)  # type: ignore[misc]
```

Comment:
mypy is correct here, `type` isn't in `__slots__` so this is an error (except in the case when subclasses have `__dict__` or `__slots__ = ('type',)` themselves -- the former seems to happen for every subclass I can find which makes `__slots__` kinda useless) -- ignoring this is the easiest fix to move forward here